sktime/forecasting/model_selection/_split.py

```diff
@@ -996,12 +996,26 @@ def get_n_splits(self, y: Optional[ACCEPTED_Y_TYPES] = None) -> int:
         n_splits : int
             The number of splits.
         """
+        from sktime.datatypes import check_is_scitype
+
         if y is None:
             raise ValueError(
                 f"{self.__class__.__name__} requires `y` to compute the "
                 f"number of splits."
             )
-        return len(self.get_cutoffs(y))
+
+        # n_splits based on the first instance of the lowest level series cutoffs
+        if check_is_scitype(y, scitype="Hierarchical"):
```

Comment:
this should also cover `"Panel"`, which is separate (e.g., 2-level `MultiIndex`)

Reply:
same as below. Kindle review them again to check if it's ok